src/Licensing/AuditThroughputCollector.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Particular.License
+{
+    using Microsoft.Extensions.Logging;
+
+    class AuditThroughputCollector
+    {
+        public AuditThroughputCollector(string serviceControlAPI, ILogger logger)
```

Comment:
I think we should use "typed" loggers wherever possible

```suggestion
        public AuditThroughputCollector(string serviceControlAPI, ILogger<AuditThroughputCollector> logger)
```

Reply:
If we are passing in the Ilogger from the main service then it will always be the type of that main service. That is typed - only way to get around that is to pass the loggerfactory around instead and i'm not sure we want to do that. Can easily be reviewed later on.

Reply:
With changing to 3 services, there will be an Ilogger per service